dev/dll/Microsoft.UI.Xaml.vcxproj.filters

```diff
@@ -116,6 +116,51 @@
     <Page Include="@(PageRequiringCustomCompilation)" />
     <Page Include="$(OutDir)rs1_compact_themeresources.xaml" />
     <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
+    <Page Include="@(PageRequiringCustomCompilation)" />
```

Comment:
I recall this being discussed in previous PRs, but I still don't understand why this line gets repeated so many times...what's causing this?  Is it a bug in VS?

Reply:
It's something in VS. I don't know what thing I do in the UI that causes it to happen though, otherwise I'd file a bug on VS.

---
In reply to: [291288068](https://github.com/microsoft/microsoft-ui-xaml/pull/819#discussion_r291288068) [](ancestors = 291288068)

Reply:
I reported this to VS very long time ago but they would not fix it.

Reply:
@mluparu FYI